autoload/jedi.vim

```diff
@@ -190,6 +190,9 @@ function! jedi#_vim_exceptions(str, is_eval)
     try
         if a:is_eval
             let l:result.result = eval(a:str)
+            if type(l:result.result) != type('')
+                let l:result.result = string(l:result.result)
+            endif
         else
```

Comment:
It took me a while to actually understand what happens here. (Because of all the context that is missing in this file). I also haven't read a lot of your ingenious error management :) It looks really good! And I think this is necessary as well.

Would you mind adding a comment? I think it can be confusing, because there's also no comment in `vim_eval` that it returns always a string :)